doc/doxygen/references.bib

```diff
@@ -26,6 +26,23 @@ @article{Kel74
   pages =   {101--129}
 }
 
+%-------------------------------------------------------------------------------
+% Step 7
+%-------------------------------------------------------------------------------
+
+@article{Li2019,
+  doi = {10.1007/s10915-019-01102-1},
+  url = {https://doi.org/10.1007/s10915-019-01102-1},
+  year = {2019},
+  month = dec,
+  publisher = {Springer Science and Business Media {LLC}},
+  volume = {82},
+  number = {1},
+  author = {Hao Li and Xiangxiong Zhang},
+  title = {Superconvergence of {\textdollar}{\textdollar}C{\^{}}0{\textdollar}{\textdollar}C0-{\textdollar}{\textdollar}Q{\^{}}k{\textdollar}{\textdollar}Qk Finite Element Method for Elliptic Equations with Approximated Coefficients},
```

Comment:
```suggestion
  title = {Superconvergence of $C^0-Q^k$ Finite Element Method for Elliptic Equations with Approximated Coefficients},
```
This looks odd. My suggestion is based on 'Brenner 2005` later in that document.